index.js

```diff
@@ -3,16 +3,51 @@ const header = document.querySelector("#header")
 console.log("Here's your header:", header)
 
 
-/***** Deliverable 2 *****/
 
+/***** Deliverable 2 *****/
 
+header.style.color = "teal"
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+console.log('This is what the traveler object looks like: ', traveler)
+
+let profileImg = document.querySelector("#profile img")
+let profileName = document.querySelector("#profile h2")
+let profileNickname = document.querySelector("#profile em")
 
+profileImg.src = traveler.photo
+profileImg.alt = "Raffy Pic"
+profileName.innerHTML = traveler.name
+profileNickname.innerHTML = traveler.nickname
 
 /***** Deliverable 4 *****/
 
+traveler.animalSightings.forEach( function (sighting) {
+    // Create elements
+
+    let li = document.createElement("li")
+    let p = document.createElement("p")
+    let img = document.createElement("img")
+    let a  = document.createElement("a")
+
+    // Add attributes and properties
 
+    li.setAttribute("data-id", sighting.id)
```

Comment:
For dataset attributes, you can also use this syntax:

```js
li.dataset.id = sighting.id
```

This will add a `data-id` attribute to the element, like this:

```html
<li data-id="1">...</li>
```